core/foundation/src/TClassEdit.cxx

```diff
@@ -1419,30 +1419,30 @@ bool TClassEdit::IsStdClass(const char *classname)
 {
    classname += StdLen( classname );
    if ( strcmp(classname,"string")==0 ) return true;
-   if ( strncmp(classname,"bitset<",strlen("bitset<"))==0) return true;
+   if ( strncmp(classname,"bitset<",std::char_traits<char>::length("bitset<"))==0) return true;
```

Comment:
```suggestion
   using std::char_traits<char>::length;

   if ( strncmp(classname,"bitset<",length("bitset<"))==0) return true;
```
Personally, I would prefer having a `using` statement at the beginning of `IsStdClass()` to make this less verbose.